tests/test_imageCollection_func.py

```diff
@@ -278,7 +279,7 @@ def test_image_collection_group_by_with_geometry_where_clause_and_multiple_alias
     sql = "select min(slp) as min_slp, avg(slp) as average_slp from 'NCEP_RE/sea_level_pressure'  group by system:index limit 5"
     response = SQL2GEE(JsonSql(sql).to_json(), geojson=r).response()
     assert len(response) == 5
-    assert len(response[0].keys()) == 5
+    assert len(response[0].keys()) == 2
```

Comment:
BC break

Reply:
as per conversation on slack, it's now clearer that 2 is probably the most adequate length here, so please disregard my comment on this line